app/src/main/java/com/sun/makeupwindow/data/ProductDataSource.kt

```diff
@@ -6,7 +6,8 @@ import com.sun.makeupwindow.data.source.utils.OnDataLoadCallback
 interface ProductDataSource {
     interface Local{
         fun getProducts(callback: OnDataLoadCallback<List<Product>>)
-        fun getItemProduct(idproduct: Int, callback: OnDataLoadCallback<Product>)
+        fun getProductIdAfterLast(callback: OnDataLoadCallback<Int>)
+        fun getProductId(idproduct: Int, callback: OnDataLoadCallback<Product>)
```

Comment:
```suggestion
        fun getProductId(id: Int, callback: OnDataLoadCallback<Product>)
```
hàm  này  truyền  vào  id  và lấy về 1 id mới à

Reply:
hàm này truyền vào id và lấy ra 1 thông tin của id đó